awscli/clidocs.py

```diff
@@ -716,3 +730,62 @@ def _line_has_tag(self, line):
 
     def doc_subitems_start(self, help_command, **kwargs):
         pass
+
+
+class GlobalsRenderer:
+    """
```

Comment:
For docstrings, we generally try to follow PEP257 specifically for [multiline doc strings](https://peps.python.org/pep-0257/#multi-line-docstrings) where the first line should be a summary of the utility and then any other elaboration can be done in a separate summary below. If I need to double check my docstring, I've had success with [`pydocstyle`](http://www.pydocstyle.org/en/stable/) in the past for quick checks.